src/test/java/teammates/test/cases/common/TimeHelperTest.java

```diff
@@ -82,6 +82,40 @@ public void testCombineDateTime() {
     }
     
     @Test
+    public void testIsWithinHour() {
+        Calendar timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        
+        ______TS("Time within past hour");
+        
+        timeCalendar.add(Calendar.MINUTE, -10);
+        assertTrue(TimeHelper.isWithinPastHour(timeCalendar.getTime(),
+                   Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+
+        ______TS("End time not within past hour");
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, -70);
+        assertFalse(TimeHelper.isWithinPastHour(timeCalendar.getTime(),
+                    Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, 10);
+        assertFalse(TimeHelper.isWithinPastHour(timeCalendar.getTime(),
+                    Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
+        
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, 10);
+        assertFalse(TimeHelper.isWithinPastHour(timeCalendar.getTime(), timeCalendar.getTime()));
+
+        ______TS("Session ended but grace time left");
+        int gracePeriod = 15;
+        timeCalendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeCalendar.add(Calendar.MINUTE, -10);
+        assertFalse(TimeHelper.isWithinPastHour(new Date(timeCalendar.getTime().getTime() + gracePeriod * 60000),
+                    Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime()));
```

Comment:
These test cases can be improved. specifically, you need to test boundary values. @tarun14110 Please open a new issue for it.